disco_aws_automation/disco_rds.py

```diff
@@ -409,12 +413,19 @@ def modify_db_parameter_group(self, db_parameter_group_name, parameters):
                                                                'MinimumEngineVersion': 'someversion',
                                                                'ApplyMethod': 'pending-reboot'}])
 
-    def recreate_db_parameter_group(self, database_name, db_parameter_group_name,
+    def recreate_db_parameter_group(self, env_name, database_name, db_parameter_group_name,
```

Comment:
`custom_db_params = custom_config.items(env_name)`

`env_name` might be different than `self.vpc_name` because I need it to read the config for the source database but create the parameter group in the current environment. For example, read `ci-txdb` but create `sandbox1-txdb`